### ISSUE TITLE
Build fails: pci.c:823:3: error: ‘new_skb’ undeclared

### ISSUE THREAD

**Issue by emilis** [lwfinger/rtlwifi_new#20]
``` bash
emilis@emilis-laptop:~/work/rtlwifi_new$ make

make -C /lib/modules/3.16.0-28-generic/build M=/home/emilis/work/rtlwifi_new modules
make[1]: Entering directory '/usr/src/linux-headers-3.16.0-28-generic'
  CC [M]  /home/emilis/work/rtlwifi_new/pci.o
/home/emilis/work/rtlwifi_new/pci.c: In function ‘_rtl_pci_rx_interrupt’:
/home/emilis/work/rtlwifi_new/pci.c:823:3: error: ‘new_skb’ undeclared (first use in this function)
   new_skb = dev_alloc_skb(rtlpci->rxbuffersize);
   ^
/home/emilis/work/rtlwifi_new/pci.c:823:3: note: each undeclared identifier is reported only once for each function it appears in
/home/emilis/work/rtlwifi_new/pci.c:827:4: error: label ‘no_new’ used but not defined
    goto no_new;
    ^
scripts/Makefile.build:257: recipe for target '/home/emilis/work/rtlwifi_new/pci.o' failed
make[2]: *** [/home/emilis/work/rtlwifi_new/pci.o] Error 1
Makefile:1345: recipe for target '_module_/home/emilis/work/rtlwifi_new' failed
make[1]: *** [_module_/home/emilis/work/rtlwifi_new] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-3.16.0-28-generic'
Makefile:43: recipe for target 'all' failed
make: *** [all] Error 2
```

Seems that the error was introduced with the last commit( 723b96cceb5614d5742b22e2edb4f0726299d282 ).

**Comment by lwfinger** [comment]
Fixed.

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
